docs/swagger/swagger.yaml

```diff
@@ -5433,6 +5433,13 @@ paths:
         required: true
         schema:
           type: string
+      - description: When set, the API will return the record that this item references
+          in its 'ref' field
```

Comment:
Pretty sure the field is "reference" spelled out (maybe this was copy-pasted from another inaccurate field description though?)